apps/ejabberd/src/ejabberd_router.erl

```diff
@@ -326,9 +415,10 @@ compile_routing_module() ->
 
 default_routing_modules() ->
     [mongoose_router_global,
-    mongoose_router_external,
-    mongoose_router_localdomain,
-    ejabberd_s2s].
+     mongoose_router_external_localnode,
+     mongoose_router_external,
+     mongoose_router_localdomain,
```

Comment:
Can we have `mongoose_router_localdomain` before external components by default? I think this is the most most plausible case when we route stanzas to other users in the system.